Production/python/scenarios.py

```diff
@@ -108,6 +108,30 @@ def __init__(self,sname):
                 era="Run2_2016",
                 localera="TM2016",
             )
+        elif sname == "Summer20UL16":
+            self.set_vars(
+                globaltag="106X_mcRun2_asymptotic_v15",
+                tagname="PAT",
+                hlttagname="HLT",
+                geninfo=True,
+                jecfile=self.absdatapath+"/jec/Summer16_07Aug2017_V11_MC",

```

Comment:
Have the JEC and JER files been updated? These seem like the old ones.

Reply:
When I initialized the new scenarios I just copied over the old paths for these. @bajgain is currently working on updating them for UL and will add the changes here.